### ISSUE TITLE
[docker] Better error message when /zap/wrk is not owned by user 'zap'

### ISSUE THREAD

**Issue by RothAndrew** [zaproxy/zaproxy#8593]
### Is your feature request related to a problem? Please describe.

If you do something like

```
docker run -it --rm \
          --network="host" \
          -v $PWD/zap/wrk:/zap/wrk:rw \
          -w /zap/wrk \
          ghcr.io/zaproxy/zaproxy:stable bash
```

but `$PWD/zap/wrk` doesn't already exist

and try to run one of the python scripts like `zap-api-scan.py`

Then inside the container `/zap/wrk` will be owned by root and zap-api-scan.py will throw a very generic an unhelpful error of "Failed to start ZAP"

### Describe the solution you'd like

I'd like it to throw a more helpful error message like "permission denied, the /zap/wrk folder is owned by root" (or something like that)

### Describe alternatives you've considered

n/a

### Would you like to help fix this issue?

- [ ] Yes

**Comment by RothAndrew** [comment]
Something else I discovered is that on my Macbook the error didn't happen. There's something different with how `docker run` behaves between Mac and Linux when it comes to mounting volumes and file/directory permissions.

**Comment by psiinon** [comment]
Yeah, it works fine on my Macbook. I dont have a linux system handy to check that..

**Comment by RothAndrew** [comment]
The first place it happened for me was in a GitHub Actions run, then confirmed on my local Rocky9 box after about 2 hours of troubleshooting 🙃
